_own_version_helper.py

```diff
@@ -13,7 +13,6 @@
 from typing import Callable
 
 from setuptools import build_meta as build_meta
-
 from setuptools_scm import Configuration
```

Comment:
This doesn't look good. We do want to separate the `setuptools_scm` imports from the rest, don't we?

Reply:
good catch - im not sure why it got merged - is the detection going wronf

Reply:
I will give [`lint.isort.known-local-folder`](https://docs.astral.sh/ruff/settings/#lint_isort_known-local-folder) a try.

Reply:
Caused by the obsolete version of ruff prior to #1096. Fixed by moving one of the commits to that PR.